app/renderer/components/config_menu.js

```diff
@@ -44,6 +44,14 @@ module.exports = {
                     <a class="waves-effect waves-light btn" v-on:click="importTasks">Import Tasks</a>
                     <label><em class="warning-text">ALERT! this will override your previous tasks</em></label>
                 </div>
+
+                <div class="shortcuts-learn">
```

Comment:
This could be a separate component (no need to fix it for now, as it is small enough to be part of config), just as an idea. What do you think?